case-lib/pipeline.sh

```diff
@@ -4,7 +4,7 @@ func_pipeline_export()
 {
```

Comment:
Typo in commit message (and PR title), should be "improve".

Reply:
Oops, Hand is faster than the brain! Thanks for the catch